ITR/data/data_warehouse.py

```diff
@@ -37,6 +37,28 @@ def __init__(self, company_data: CompanyDataProvider,
         self.column_config = column_config
         self.company_data = company_data
         self.company_data._calculate_target_projections(benchmark_projected_production)
+        
+        # After projections have been made, shift S3 data into S1S2.  If we shift before we project,
+        # then S3 targets will not be projected correctly.
+        for c in self.company_data._companies:
+            if c.ghg_s3:
+                # For Production-centric and energy-only data (except for Cement), convert all S3 numbers to S1 numbers
+                c.ghg_s1s2 = c.ghg_s1s2 + c.ghg_s3
```

Comment:
Yes, I see. It might be a cheap way to integrate automotive use-case into the system, but we will but ourselves into a big technical debt.
It seems more beneficial to me, if we could start supporting S3 correctly from the start. Refactoring these changes into "as intended" implementation will cause us a lot of efforts and troubles.

Do we need to support automotive ASAP?
Or can we take time, to do it right?

Reply:
The OECM benchmarks provide a conventional benchmark that defines S3 emissions and tails them off through 2050.  It also provides a "producer-centric" benchmark that folds S3 into S1 and treats S3 as zero.  They have constructed both definitions and implementations so that producer-centric, sales-centric, and regular all compute same end results w/o double-counting.  You can read their paper on that.

In the meantime, the benchmark data we have IS producer-centric, so we fold S3 into S1 as per methodology.  We must do target projections before folding, since S1, S2, and S3 targets may all be different.  But this is not suto-specific, but all sectors, except possibly a special case for cement (where cement S3 becomes Buildings S2).

Reply:
You might be right from high-level perspective. Unfortunately, details are too complex, for me to understand, yet.

From close perspective, I see that we set s1s2 = s1s2 + s3, which looks wrong to me.
First thought: we have s1s2s3 scope, for this situation.
Second thought: we maybe still should treat s3 as s3 alone, and teach the tool, to display s3 only via GUI.
Mixing s3 into s1s2 seems not right from both perspectives

Reply:
I see what you are saying.  We will know a lot more when we have S3-carrying benchmark data.  Too soon to tell how best to deal with very specific guidance that "S3 becomes S1 in producer-specific benchmark".

Reply:
I read through Chapter 3 of "Scope 3 Emissions Assessment and Circular Economy Protocol Development at Ford Motor Company", and I didn't find the mention of "S3 become S1" or similar. The closest reference I found is on the page 36 - Ford wants to submit "refrigerant switching and confinement initiatives" under S1, instead of S3.

Maybe you mentioned a different paper - in this case, could you give me a reference?

Mixing S3 into S1 seems to me like a wrong step. Are you sure, that we need it for the methodology? If we chose it, to overcome inability of our GUI to display S3 or S1S2S3 scopes - maybe, we should think more in this direction instead?

To be clear, I find many side changes in this PR helpful, to improve the tool. If we could extract side changes into a separate PR, they could be merged into develop easier, from my perspective. It would be not efficient, if one change, which we argue about, become a roadblock for all other improvements

Reply:
Check here: https://www.unepfi.org/publications/one-earth-climate-model-sectoral-pathways-to-net-zero-emissions/

Look for "producer-centric" in documentation embedded in XLSX spreadsheet containing benchmark data.

Reply:
In chapter 5.2 "key results", I see, that they mention, that S3 scope is double-counted in the current report practice.
In chapter 7 "data format" page 35, I see, that mixing S3 into S1 creates inconsistent data. My interpretation of page 35: to keep data consistent, they recommend to add an extra separation "production-centric" for sectors, which report double-counted data.

Mixing S3 into S1 doesn't seem like a recommended path.

You mentioned earlier, that it's not yet clear, how to implement new recommendations in the tool.
In this case, should we figure it out first, before taking actions?

Reply:
This text from row 120 of the "Data Information" Sheet of [this spreadsheet](https://www.uts.edu.au/sites/default/files/2022-04/OECM_OECD_North_America_2022_04_22_results_0.xlsx) explains:

```
Table B2 shows a   schematic representation of the OECM Scope 1, 2, and 3 calculation method according to GICS class, used to avoid double counting. The sum of Scopes 1, 2, and 3 for each of the three categories is equal to the actual emissions.

Example: Total annual global energy-related CO2 emissions are 35 Gt in a   given year.
     •	The sum of Scope 1, 2, and 3 for the primary class is 35 Gt CO2
     •	The sum of Scope 1, 2, and 3 for the secondary class is 35 Gt CO2 
    •	The sum of Scope 1, 2, and 3 for end-use activities is 35 Gt CO2

Double counting can be avoided by defining a primary class for the primary energy industry, a secondary class for the supply utilities, and an end-use class for all the economic activities that use the energy from the primary- and secondary-class companies. Furthermore, the separation of all emissions by defined industry categories—such as GICS—streamlines the accounting and   reporting systems. The volume of data required is reduced and reporting is   considerably simplified under the OECM methodology.   Achieving the global target of 1.5 °C and net-zero emissions by 2050 under the Paris Agreement for a specific industry sector, requires all its business activities with other sectors to also commit to a 1.5 °C–net-zero emissions targets.
```
When you look at the data for Scope 3 emissions in the Road Transport benchmark (Rows 117-122) on the "Transport UNPRI" sheet, you will see Scope 3 listed as zero for all years.  Contrast with the benchmark above--same information but different model definition--and you will see Scope 3 is 100x Scope 1 emissions.  The ITR tool has data based on the Producer-centric benchmark, not the Energy-centric benchmark.  But according to the documentation, both should produce the same results, they just get there in different ways.

Reply:
> When you look at the data for Scope 3 emissions in the Road Transport benchmark (Rows 117-122)

Yes, I see it. I agree, it should sum up to the same values for both energy-centric and production-centric. And it does sum up, if we pay attention to S1S2S3, instead of S1S2.
I notice, that there is only one sector (transport), which will benefit from mixing up. Other scopes don't show this pattern, as far as I can tell.
Yes, same reported emissions may be included in either S1 or S3, depending on the angle of view: energy-centric or production-centric. Is it wrong? Seems fine to me, as long as the summary S1S2S3 is the same.

Could the fix be displaying S1S2S3 in our GUI? Or have a selection in GUI, which scope aggregation user wants to see? Mixing S3 into S1S2 still seems to me like a sub-optimal option

Reply:
If you look at Energy (oil & gas) producer-centric, not only do we see scope 3 collapse to zero into scope 1, we see scope 2 collapse as well.  So there's more to fix :-)

Reply:
Maybe we need one more ITR member review, to give us a third view on this particular aspect

Reply:
@joriscram what do you think about mixing S3 into S1S2?